README.md

```diff
@@ -23,6 +23,30 @@ computers.
   </a>
 </div>
 
+## Installation
+
+See [CLI](./cli/README.md).
```

Comment:
How do you feel about linking directly to the Installation section of CLI's README?

```suggestion
See [CLI Installation](./cli/README.md#installation).
```